main.py

```diff
@@ -246,26 +249,49 @@ def match_flows(self, switch, args, stored_flows, many=True):
             return None
         return response
 
-    # pylint: disable=redefined-outer-name
-    # pylint: disable=too-many-branches
+    def process_tables(self, switch, table_id, args, stored_flows, actions):
+        """Resolve the table context and get actions in the matched flow"""
+        goto_table = False
+        actions_ = []
+        flow = self.match_flows(switch, table_id, args, stored_flows, False)
+        if flow and 'actions' in flow['flow']:
+            actions_ = flow['flow']['actions']
+        elif flow and 'instructions' in flow['flow']:
+            for instruction in flow['flow']['instructions']:
+                if instruction['instruction_type'] == 'apply_actions':
+                    actions_ = instruction['actions']
+                elif instruction['instruction_type'] == 'goto_table':
+                    table_id_ = instruction['table_id']
+                    if table_id < table_id_:
+                        table_id = table_id_
+                        goto_table = True
+                    else:
+                        msg = f"Wrong table_id: \
+                            A packet can only been directed to a \
+                                flow table number greather than {table_id}"
```

Comment:
You'll probably want to include the `flow['flow']` entry too otherwise you'll never know which flow entry matched an invalid table, otherwise you won't have enough data to fully understand how it processed a wrong table.

Reply:
Done.